app/src/main/java/com/example/ayapp/habits/MainActivity.java

```diff
@@ -0,0 +1,79 @@
+package com.example.ayapp.habits;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.ListView;
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+
+    /***********************************************************************************************
+     * VARIABLES
+     **********************************************************************************************/
+    // Create the adapter to convert the array to views
+    HabitListAdapter adapter;
+    User user;
+    Habit habit;
+    //created habit object for testing
+    //TODO: remove before pushing files
+    Habit defaultHabit =
+            new Habit("Default Habit","This is how the Habit description is shown",5,1234);
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        //Toolbar
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayShowTitleEnabled(true);
+
+        user = new User("Me","me@ayapp.com","password");
+        //add random habit to arrayOfHabits for testing
+        // TODO: remove before pushing files
+        user.addHabit(defaultHabit);
+
+        adapter = new HabitListAdapter(this, user.getHabits());
+        ListView habitListView = (ListView) findViewById(R.id.habitListView);
+        habitListView.setAdapter(adapter);
```

Comment:
you might want to make this:
`habitListView.setAdapter(new HabitListAdapter(this, user.getHabits())`